test/utils/file.test.ts

```diff
@@ -0,0 +1,36 @@
+import { expect } from 'chai';
+const fs = require("fs");
+import { getSoftwareHashedValuesFile, getStoredHashedValues, storeHashedValues } from '../../src/utils/file';
+
+describe("file util", function () {
+  beforeEach(function () {
+    deleteHashedFile();
+  });
+
+  afterEach(function () {
+    deleteHashedFile();
+  });
+
+  context("when no file exists", function() {
+    it("returns an empty hash", async function() {
+      const hashedVals = getStoredHashedValues();
+      expect(Object.keys(hashedVals).length).to.eq(0);
+    });
+
+    it("returns an hash with a file value", async function() {
```

Comment:
returns a hash

Reply:
Should this test be in another context, "when a file already exists"?

Reply:
yeah, good point. i've added that